# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/cairo/cairo/moz.yaml
# Commit: 3cdfb39376b7
# Full Hash: 3cdfb39376b736597fe658a322389117686b7e82
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2024-05-07 09:13:07
# Regressor Bug: 1892913
# File Overlap Count: 3
# Description:
#   Bug 1892913 - patch 1 - Update cairo source to 1.18.0 r=gfx-reviewers,lsalzman
#   
#   Wholesale replacement of files in gfx/cairo/cairo with contents of the 1.18.0 release tarball,
#   omitting subdirectories that are not part of the core library build.
#   
# ==============================================================================

diff -r 9daa351f875b -r 3cdfb39376b7 gfx/cairo/cairo/moz.yaml
--- a/gfx/cairo/cairo/moz.yaml	Mon May 06 19:22:35 2024 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,16 +0,0 @@
-schema: 1
-
-bugzilla:
-  product: "Core"
-  component: "Audio/Video: Playback"
-
-origin:
-  name: "cairo"
-  description: "2D graphics library"
-
-  url: "https://www.cairographics.org/"
-  license: "MPL-1.1"
-  license-file: "COPYING-MPL-1.1"
-
-  # git://anongit.freedesktop.org/git/cairo
-  release: "12d521df8acc483b2daa844d4f05dc2fe2765ba6 (2010-01-21)"